src/XR/features/WebXRDOMOverlay.ts

```diff
@@ -0,0 +1,139 @@
+import { Tools } from "../../Misc/tools";
+import { Nullable } from "../../types";
+import { WebXRFeatureName, WebXRFeaturesManager } from "../webXRFeaturesManager";
+import { WebXRSessionManager } from "../webXRSessionManager";
+import { WebXRAbstractFeature } from "./WebXRAbstractFeature";
+
+/**
+ * Options for DOM Overlay feature
+ */
+export interface IWebXRDOMOverlayOptions {
+    /**
+     * Fail the Web XR Session if DOM overlay isn't available (TODO: would like to mark optional as a feature parameter)
+     */
+    // required?: boolean;
+
+    /**
+     * DOM Element for overlay into WebXR scene.
+     *
+     * NOTE: UA may make this element background transparent in XR.
+     */
+    element: Element;
```

Comment:
wondering if a selector might not be better here?

can be `Element | string` , and if it is a string, define the element using `document.querySelector`

Reply:
yes - that is better!

Reply:
also, the comment was changed to indicate how the string works.
```
DOM Element or document query selector string for overlay.
```